scripts/add_nagim_gvcfs_to_tob_wgs_test_metamist.py

```diff
@@ -0,0 +1,143 @@
+from metamist.graphql import gql, query
+from metamist.apis import ParticipantApi, AnalysisApi
+from metamist.models import (
+    ParticipantUpsert,
+    SampleUpsert,
+    SequencingGroupUpsert,
+    AssayUpsert,
+    AnalysisStatus,
+    Analysis,
+)
+from cpg_utils import to_path
+import csv
+
+import click
+
+PARTICIPANT_QUERY = gql(
+    """
+    query ($project: String!) {
+        project(name: $project) {
+            id
+            participants {
+            externalId
+            id
+            samples {
+                id
+                externalId
+                sequencingGroups {
+                externalIds
+                id
+                meta
+                platform
+                technology
+                type
+                assays {
+                    meta
+                    id
+                    }
+                }
+            }
+        }
+    }
+}
+"""
+)
+
+
+@click.command()
+@click.option(
+    '--project',
+    required=True,
+    help='The name of the project to add samples to.',
+)
+@click.option(
+    '--project_id',
+    required=True,
+    type=int,
+    help='The ID of the project to add samples to.',
+)
+@click.option(
+    '--new_samples_path',
+    required=True,
+    help='The path to the file containing the newly created samples.',
```

Comment:
Can you be more descriptive to what this means? Including the expected structure of the file, and maybe an anonymised example of the first couple of lines?

Reply:
I changed the parameter name because it doesn't actually reflect what we're parsing, here's the new:
```python
@click.option(
    '--sample-path-mappings',
    required=True,
    help='''The path to a CSV file containing mappings of `main` CPG ID's, 
            the `external_id` and `gvcf` paths. 
            The file should have at least four columns: sgid, ext_id, gvcf, and gvcf_idx. 
            Here's an example of what the first couple of lines might look like:

            sgid,ext_id,gvcf,gvcf_idx
            sg1,ext1,gvcf1,gvcf_idx1
            sg2,ext2,gvcf2,gvcf_idx2
        '''
)
```